apps/backend/src/lambdas/parser.ts

```diff
@@ -10,7 +10,7 @@ export const handler = async (event: S3Event, _context?: Context) => {
   appLogger.setContext('Parser Lambda');
   const parseService = app.get(ParseService);
   appLogger.log({ event, _context }, 'PARSE EVENT');
-  const automatedReconciliationEnabled = !process.env.DISABLE_LOCAL_RECONCILE;
+  const automatedReconciliationEnabled = process.env.RUNTIME_ENV === 'test';
```

Comment:
What risks exist here - does this get set to "dev" or "prod" in those environments? Did we fully deprecate this var or could someone still try setting it and get unexpected behaviour?

Reply:
I have only set it to run on test currently as we don't have the prod env up yet. There is probably a better way to handle this one... we don't want it to run on local, and I've disabled it for dev so that Qudsia can run batch reconciliation on that env instead... maybe this warrants a discussion? Basically, if the sync job is running, or if we're in our production env, we want to enable automation, if the sync job is disabled, then we should run the data in batches...

Reply:
I guess this should be in another ticket. The functionality is the same, I was just trying to provide clarity on where/when this should be disabled.

Reply:
Well we're not in prod yet, so I'm thinking we don't need to effectively have a feature flag disabling this in prod right now? Otherwise we'll need to change this code again in order to perform the release unless I'm missing something.

Reply:
This is meant to disable on dev. I see how I made it confusing. One moment I will fix it.

Reply:
![Screenshot 2023-08-22 at 10 26 12 AM](https://github.com/bcgov/PaymentCommonComponent/assets/78890675/5cceda9f-0ab5-400b-a352-072d3b78dba0)

![Screenshot 2023-08-22 at 10 26 40 AM](https://github.com/bcgov/PaymentCommonComponent/assets/78890675/67752606-5112-40dc-9dd5-4bdfde4a0c71)

Reply:
Setting it in the makefile to be more configurable...